### PR TITLE
add NFT discovery section

### PR DESCRIPTION
**Pull request by kdembler**
Changes:
1. Added "New NFTs on sale" section on homepage
2. Added "Video NFTs" view with all the NFTs and filtering/pagination
3. Made NFT tile's full area clickable, not only the thumbnail
4. Changed NFT status filtering to use checkboxes again (you can select multiple statuses at once)

### CODE REVIEW

**Comment by toiletgranny** [comment]
Perfect! As far as I can see — and I checked all the filters and counted the results — everything seems to be working fine!

The only thing that's bothering me is why can't we display the number of selected checkboxes in the blue badge in the "Status" filter group when the "Other filters" group manages this just fine? Not a blocker for me, though, in case that's a bigger issue.

**Review by toiletgranny — APPROVED** [review]

**Comment by kdembler** [comment]
> The only thing that's bothering me is why can't we display the number of selected checkboxes in the blue badge in the "Status" filter group when the "Other filters" group manages this just fine? Not a blocker for me, though, in case that's a bigger issue.

I just missed that when changing from radio button. Fixed now

File: packages/atlas/src/views/viewer/NftsView/NftsView.tsx [hunk 1]
```diff
@@ -0,0 +1,164 @@
...
+  const smMatch = useMediaMatch('sm')
+  const mdMatch = useMediaMatch('md')
+
+  const filtersBarLogic = useFiltersBar()
+  const {
+    filters: { setIsFiltersOpen, isFiltersOpen },
+    canClearFilters: { canClearAllFilters },
+    ownedNftWhereInput,
+    videoWhereInput,
+  } = filtersBarLogic
```
Comment (drillprop) [inline packages/atlas/src/views/viewer/NftsView/NftsView.tsx:38]: Hmmm, now we could apply the same filters on MemberView and ChannelView. Right now we can only filter by status there. 
Just saying, let's do this as a follow-up maybe?

**Review by drillprop — APPROVED** [review]
That's a big improvement, now NFTs are easier to find. That feels much better.

image

File: packages/atlas/src/views/viewer/NftsView/NftsView.tsx (hunk 1 above)
Comment (kdembler) [reply packages/atlas/src/views/viewer/NftsView/NftsView.tsx:38]: Yeah, let's do that later

**Pull request merged**